domain-rrd/rrd-shared/src/main/java/gov/va/vro/model/rrd/mas/MasAutomatedClaimPayload.java

```diff
@@ -95,7 +96,8 @@ public String getDisabilityActionType() {
    */
   @JsonIgnore
   public boolean isInScope() {
-    return Objects.equals(getDiagnosticCode(), BLOOD_PRESSURE_DIAGNOSTIC_CODE)
+    return (Objects.equals(getDiagnosticCode(), HYPERTENSION_DIAGNOSTIC_CODE)
```

Comment:
Why does it use Objects.equals instead of String equals?

Reply:
I do not know, we can ask @alan-amida